src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-react/jsx-space-before-trailing-slash.ts

```diff
@@ -0,0 +1,12 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertJsxSpaceBeforeTrailingSlash: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "react/jsx-space-before-closing",
```

Comment:
Should be `react/jsx-tag-spacing`.

Reply:
I was too excited to finally tackle these. 😄